app/assets/javascripts/shared/time/timeDirective.spec.js

```diff
@@ -39,12 +59,6 @@ describe('ocTime', function () {
       expect(directiveElem.text()).toBe('02:03');
     });
 
-    it('should update time', function () {
```

Comment:
Why remove this test? Instead it should be fixed to work with the new behaviour.